controller/src/main/java/io/pravega/controller/server/rest/v1/ApiV1.java

```diff
@@ -146,6 +146,24 @@ void deleteStream(@ApiParam(value = "Scope name", required = true) @PathParam("s
                 @ApiParam(value = "Stream name", required = true) @PathParam("streamName") String streamName,
                 @Context SecurityContext securityContext, @Suspended final AsyncResponse asyncResponse);
 
+        @DELETE
+        @Path("/{scopeName}/readergroups/{readerGroupName}")
+        @io.swagger.annotations.ApiOperation(value = "", notes = "Delete a reader group", response = void.class,
+                tags = { "ReaderGroups", })
+        @io.swagger.annotations.ApiResponses(value = {
+                @io.swagger.annotations.ApiResponse(code = 204, message = "Successfully deleted the reader group",
+                        response = void.class),
+
+                @io.swagger.annotations.ApiResponse(code = 404, message = "Scope or reader group with given name not found",
```

Comment:
is it possible to segregate both? One for scope not found and another for readergroup not found?